tests/unit_tests/services/test_dds.py

```diff
@@ -135,6 +135,9 @@ def _get_delivery_order():
                         ]),
                     ])
 
+        dds_version = dds_project.get_dds_version()
+        assert dds_version
+
```

Comment:
I think it would be better to have a new test for that, where you mock `subprocess.run` (or `ExternalProgramService` if you choose to use `_run` instead of `subprocess.run`) to return a something like `Data Delivery System, version 2.6.1`, and then assert that `get_dds_version` returns `2.6.1`.